mass-stop-start-hana-cloud/README.md

```diff
@@ -0,0 +1,65 @@
+
+## Requirements
+
+To use this example you'll need the following:
+
+* One or more HANA Cloud databases in the Cloud Foundry environment
+* Instance of [SAP Service Manager](https://help.sap.com/docs/service-manager/sap-service-manager/sap-service-manager) with plan *subaccount-admin* and a service key/bindin with default configurations.
+* Platform user with *Space Developer* role in the spaces where the databases reside
+
+:warning: If you are planning to use SAP Universal ID, please keep [SAP Note 3085908](https://launchpad.support.sap.com/#/notes/3085908) in mind
+
+Check out the following resources for more information:
+
+* [Deploy SAP HANA Cloud](https://developers.sap.com/tutorials/hana-cloud-deploying.html)
+* [Create Users and Manage Roles and Privileges](https://developers.sap.com/tutorials/hana-cloud-mission-trial-4.html)
+* [Creating Service Instances in Other Environments](https://help.sap.com/docs/service-manager/sap-service-manager/creating-instances-in-other-environments)
+* [Creating Service Bindings in Other Environments](https://help.sap.com/docs/service-manager/sap-service-manager/creating-service-bindings-in-other-environments)
+
+## How to use
+
+Import the content of [examples catalog](catalog.json) in your Automation Pilot tenant.
+The examples includes two commands - *MassStopHanaCloudInstances* and *MassStartHanaCloudInstances*. Navigate to any of them and to trigger them.
+
+You'll need to provide values for the following input keys:
+
+* *serviceKey* - Service Key for SAP Service Manager
+* *user* - Email or ID of your technical user
+* *password* - Password of your technical user
+* *identityProvider* - Optional: origin key of your identity provider. Defaults to sap.ids
+
+As mentioned before, Automation Pilot allows executions to be automatically triggered on regular intervals - hourly, daily, weekly, monthly or yearly. We can create two [Scheduled Executions](https://help.sap.com/docs/AUTOMATION_PILOT/de3900c419f5492a8802274c17e07049/96863a2380d24ba4bab0145bbd78e411.html) - one to stop all databases after working hours and another to start them up again on the next morning.
+
+Here are the some example schedule configurations:
+
+* *MassStopHanaCloudInstances* - executed every weekday at 16:30 UTC
+  * Schedule - Weekly on Monday, Tuesday, Wednesday, Thursday and Friday
+  * Hours - 16
+  * Minutes - 30
+* *MassStartHanaCloudInstances* - every weekday at 06:30 UTC
+  * Schedule - Weekly on Monday, Tuesday, Wednesday, Thursday and Friday
+  * Hours - 6
+  * Minutes - 30
+
+Another important aspect is alerting. It's important to receive notifications (in the form of email, slack message, Jira ticket or other) whenever the commands fail. This could be easily achieved with the help of SAP Alert Notification. More information can be found [here](https://help.sap.com/docs/AUTOMATION_PILOT/de3900c419f5492a8802274c17e07049/e75533639c6d4193aa8a7e7420c25f8c.html).
```

Comment:
"... (or use any other alerting channel supported , you can find more within the [Alert Notification documentation here](https://help.sap.com/docs/ALERT_NOTIFICATION/5967a369d4b74f7a9c2b91f5df8e6ab6/eaaa37e6ff62486ebb849507dc33abc6.html))"